vulture/config.py

```diff
@@ -0,0 +1,168 @@
+"""
+This module handles retrieval of configuration values from either the
+command-line arguments or the pyproject.toml file.
+"""
+# pylint: disable=bad-continuation
+import argparse
+from os.path import exists
+from typing import Any, Dict, List, Optional
+from typing.io import TextIO
+
+import toml
+
+from .version import __version__
+
+MIN_CONFIDENCE_DEFAULT = 0
+
```

Comment:
I think we should add a list or dict of option names and their defaults. Then we can check that the TOML file has no extraneous keys and can loop over the list when creating Config objects instead of listing all option names twice.

Reply:
It would be good to specify the defaults only once, otherwise they might go out of sync. But maybe we also only need to specify them once, because the CLI options override the TOML options anyway.

Reply:
Addressed in 9b19a1a